index.html

```diff
@@ -271,20 +271,20 @@ <h3>LET'S CONNECT</h3>
                             <li> 
                                 <div class="flex">
                                     <p>Languages</p>
```

Comment:
- [ Optional ] Kindly add some padding to the `Language, Frameworks `and `Skills` paragraph. Besides, what do you think if you can use heading tags (`h3, h4, ..`) instead of `p` since they are headings naturally and it will be semantically meaningful?
![skills-section-p-padding](https://user-images.githubusercontent.com/44057863/167713711-7abac407-9410-41f6-b186-0d9edb84ce4d.png)